<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MsaExportSettings</class>
 <widget class="QWidget" name="MsaExportSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Region</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Whole alignment</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Custom region</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectRegionButton">
       <property name="maximumSize">
        <size>
         <width>65</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="exportSeqNames">
     <property name="text">
      <string>Include names</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="exportConsensus">
     <property name="text">
      <string>Include consensus</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="exportRuler">
     <property name="text">
      <string>Include ruler</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="multilineModeCheckbox">
       <property name="text">
        <string>Bases per line (multiline image):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="multilineWidthSpinbox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>multilineModeCheckbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>multilineWidthSpinbox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
